tests/shared/src/test/scala/scala/meta/tests/parsers/DeclSuite.scala

```diff
@@ -6,23 +6,30 @@ import scala.meta.dialects.Scala211
 
 class DeclSuite extends ParseSuite {
   test("val x: Int") {
-    val Decl.Val(Nil, List(Pat.Var(Term.Name("x"))), Type.Name("Int")) = templStat("val x: Int")
+    assertTree(Decl.Val(Nil, List(Pat.Var(Term.Name("x"))), Type.Name("Int")))(
```

Comment:
i thought the first parameter is "obtained" and the second is "expected", i.e. swap them? not at the computer right now, but could check the parameter names to assertTree when I get to it.

Reply:
Ach right, I tried automating the changes a bit, so this was done wrong. Fixed now!